FiveCalls/FiveCalls/en.lproj/Main.strings

```diff
@@ -14,14 +14,14 @@
 /* Class = "UIButton"; normalTitle = "Set Location"; ObjectID = "6mh-1s-XbV"; */
 "6mh-1s-XbV.normalTitle" = "Set Location";
 
-/* Class = "UILabel"; text = "Or Enter Your Zip Code"; ObjectID = "7Xf-pd-kKj"; */
-"7Xf-pd-kKj.text" = "Or Enter Your Zip Code";
+/* Class = "UILabel"; text = "Or Enter Your Zip Code or Address"; ObjectID = "7Xf-pd-kKj"; */
+"7Xf-pd-kKj.text" = "Or Enter Your Zip Code or Address";
```

Comment:
I think this would read better as:

`Or enter manually:`

(Can you also increase the spacing between the "Use My Location" button and this label?)

![iphone 7 ios 10 2 14c89 2017-02-25 14-19-09](https://cloud.githubusercontent.com/assets/59140/23334483/70acf1fc-fb65-11e6-89a7-dc72a81b21ae.png)